script.js

```diff
@@ -79,6 +79,37 @@ const cancelEditingTodoHandler = (
 
     editButton.innerText = "Edit";
     todo.isEditing = false;
+    todo.isComplete = false;
+};
+
+const markDoneTodoHandler = (
+    e,
+    paragraphElement,
+    editButton,
+    inputElement,
+    cancelButton,
+    todo
+) => {
+    if (!inputElement.value) {
```

Comment:
```suggestion
    if (!inputElement.value.trim()) {
```
Let's use trim() here, which ignores adding whitespaces or checks non empty content more effectively.